core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -233,6 +229,69 @@ private[ui] class ExecutorsPage(
       </tbody>
     </table>
   }
+
+  private def taskData(maxTasks: Int,
+                        activeTasks: Int,
```

Comment:
can you change the spacing on all the parameters to be 4 spaces indented.  Put the first parameter on a newline.

Basically keep it the same as the constructor in this file private[ui] class ExecutorsPage(